cli/main.go

```diff
@@ -296,6 +296,14 @@ func beforeSubcommands(c *cli.Context) error {
 		ignoreConfigLogs = true
 	} else {
 		if path := c.GlobalString("log"); path != "" {
+			// avoid log.json file too large consume the memory footprint in the tmpfs,
+			// truncate the log.json file every time before kata subcommand is executed
+			if path != "/dev/null" && katautils.FileExists(path) {
```

Comment:
please don't do this, instead check if path is a regular file and if it exists

Reply:
😀 could you give me some advice? thanks.